dev-tools/reconfigurator-cli/tests/output/cmds-target-release-stdout

```diff
@@ -8049,20 +8052,232 @@ to:   blueprint 4713f6c4-e8ba-4a28-87a0-df75ebf7b8b6
+-       SRV  port  5353 6c3ae381-04f7-41ea-b0ac-74db387dbc3a.host.control-plane.oxide.internal
+-       SRV  port  5353 803bfb63-c246-41db-b0da-d3b87ddfc63d.host.control-plane.oxide.internal
+-       SRV  port  5353 f6ec9c67-946a-4da3-98d5-581f72ce8bf0.host.control-plane.oxide.internal
++       SRV  port  5353 26bdd109-c842-43a9-95cb-15aba9b0832b.host.control-plane.oxide.internal
++       SRV  port  5353 4ab0ec67-b27e-42b5-af22-9117ad11113b.host.control-plane.oxide.internal
++       SRV  port  5353 63ff80e3-ef8d-4186-8240-5ebd3f7d4d82.host.control-plane.oxide.internal
+*   name: _internal-ntp._tcp                                 (records: 3 -> 3)
+-       SRV  port   123 62620961-fc4a-481e-968b-f5acbac0dc63.host.control-plane.oxide.internal
+-       SRV  port   123 6444f8a5-6465-4f0b-a549-1993c113569c.host.control-plane.oxide.internal
+-       SRV  port   123 f10a4fb9-759f-4a65-b25e-5794ad2d07d8.host.control-plane.oxide.internal
++       SRV  port   123 9e2e0774-3cf6-4f75-9a12-92db05c77b81.host.control-plane.oxide.internal
++       SRV  port   123 ba87399e-e9b7-4ee4-8cb7-0032822630e9.host.control-plane.oxide.internal
++       SRV  port   123 e14f91b0-0c41-48a0-919d-e5078d2b89b0.host.control-plane.oxide.internal
+*   name: _nameservice._tcp                                  (records: 3 -> 3)
+-       SRV  port  5353 427ec88f-f467-42fa-9bbb-66a91a36103c.host.control-plane.oxide.internal
+-       SRV  port  5353 99e2f30b-3174-40bf-a78a-90da8abba8ca.host.control-plane.oxide.internal
+-       SRV  port  5353 ea5b4030-b52f-44b2-8d70-45f15f987d01.host.control-plane.oxide.internal
++       SRV  port  5353 698d1d82-0620-4978-93ac-0ba5d40f3da9.host.control-plane.oxide.internal
++       SRV  port  5353 abd27551-4027-4084-8b52-13a575b035b4.host.control-plane.oxide.internal
++       SRV  port  5353 d5fd048a-8786-42d3-938e-820eae95d7f4.host.control-plane.oxide.internal
++   name: abd27551-4027-4084-8b52-13a575b035b4.host          (records: 1)
++       AAAA fd00:1122:3344:2::1
+-   name: ad6a3a03-8d0f-4504-99a4-cbf73d69b973.host          (records: 1)
+-       AAAA fd00:1122:3344:102::25
+-   name: ba4994a8-23f9-4b1a-a84f-a08d74591389.host          (records: 1)
+-       AAAA fd00:1122:3344:101::24
++   name: ba87399e-e9b7-4ee4-8cb7-0032822630e9.host          (records: 1)
++       AAAA fd00:1122:3344:102::29
++   name: d5fd048a-8786-42d3-938e-820eae95d7f4.host          (records: 1)
++       AAAA fd00:1122:3344:3::1
++   name: e14f91b0-0c41-48a0-919d-e5078d2b89b0.host          (records: 1)
++       AAAA fd00:1122:3344:101::28
+-   name: ea5b4030-b52f-44b2-8d70-45f15f987d01.host          (records: 1)
+-       AAAA fd00:1122:3344:3::1
+-   name: f10a4fb9-759f-4a65-b25e-5794ad2d07d8.host          (records: 1)
+-       AAAA fd00:1122:3344:103::21
+-   name: f6ec9c67-946a-4da3-98d5-581f72ce8bf0.host          (records: 1)
+-       AAAA fd00:1122:3344:103::23
+    unchanged names: 35 (records: 41)
```

Comment:
The internal DNS diff is basically all noise.  What we care about here is *not* seeing `_nexus._tcp` having changed.